qi-lib/flow/core/deforest.rkt

```diff
@@ -180,17 +180,17 @@
   ;; producer. Procedures accepting variable number of arguments like
   ;; `map` cannot be in this class.
   (define-syntax-class fusable-stream-transformer0
-    #:attributes (f next)
```

Comment:
Why did these attributes disappear from this class?

I'm also confused about why filter/filter-map can be here if map cannot
(filter-map is vararg), especially when reckoning with "head position of the
fused sequence even when there is no explicit producer." Why can filter be at
the head position without an explicit producer?

Reply:
Because core.rkt module is only used for matching, not for producing any new code with syntax templates and therefore the attributes of any potential template variables are not used. The attributes used are declared in the syntax classes within the cps.rkt module.